ingest/validation/validate_metadata.py

```diff
@@ -342,15 +359,19 @@ def retrieve_ontology_term(convention_url, ontology_id):
     """
     OLS_BASE_URL = "https://www.ebi.ac.uk/ols/api/ontologies/"
     convention_ontology = retrieve_ontology(convention_url)
+    # separate ontology shortname from term ID number
+    # valid separators are underscore and colon (used by HCA)
     try:
-        ontology_shortname, term_id = ontology_id.split('_')
+        ontology_shortname, term_id = re.split('[_:]', ontology_id)
+    # when ontolgyID is malformed and has no separator -> ValueError
     except ValueError as error:
         print("ValueError:", error)
-        print('missing ontology shortname in', ontology_id)
+        print('Problem with provided ontologyID', ontology_id)
```

Comment:
If "ontologyID" is a term the user would see, I'd suggest adding a space to make it read more conventionally -- i.e.:
```suggestion
        print('Problem with provided ontology ID', ontology_id)
```

And similarly for other user-facing instances of the term.

Reply:
updated in `4023a08`